go.mod

```diff
@@ -4,7 +4,7 @@ go 1.22.8
 
 require (
 	github.com/docker/docker v26.1.5+incompatible
-	github.com/elastic/elastic-agent-libs v0.9.13
+	github.com/elastic/elastic-agent-libs v0.17.3-0.20241112062438-5ba501c7ca8c
```

Comment:
This will be updated once https://github.com/elastic/elastic-agent-libs/pull/252 is merged and released